engine/src/main/java/org/terasology/rendering/cameras/PerspectiveCamera.java

```diff
@@ -49,7 +51,8 @@
 
     private Vector3f tempRightVector = new Vector3f();
 
-    public PerspectiveCamera(PerspectiveCameraSettings cameraSettings) {
+    public PerspectiveCamera(PerspectiveCameraSettings cameraSettings, WorldProvider worldProvider, RenderingConfig renderingConfig) {
```

Comment:
The cameraSettings can be obtained via renderingConfig.getCameraSettings() - no need to pass them in explicitly anymore.